src/Compilers/CSharp/Portable/Lowering/SynthesizedMethodBaseSymbol.cs

```diff
@@ -117,7 +117,13 @@ private ImmutableArray<ParameterSymbol> MakeParameters()
             var parameters = this.BaseMethodParameters;
             foreach (var p in parameters)
             {
-                builder.Add(SynthesizedParameterSymbol.Create(this, this.TypeMap.SubstituteType(p.OriginalDefinition.TypeWithAnnotations), ordinal++, p.RefKind, p.Name));
+                builder.Add(SynthesizedParameterSymbol.Create(
+                    this,
+                    this.TypeMap.SubstituteType(p.OriginalDefinition.TypeWithAnnotations),
+                    ordinal++,
+                    p.RefKind,
+                    p.Name,
+                    attributes: p.GetAttributes()));
```

Comment:
>p.GetAttributes() [](start = 32, length = 17)

It doesn't look like this is an appropriate thing to do for BaseMethodWrapperSymbol. Please add a test for a scenario like that. It would probably be better if we had an explicit flag indicating whether attributes should be "inherited". Or perhaps we can simply check if the ```BaseMethod``` is a local function.